src/mumble/Settings.cpp

```diff
@@ -1006,6 +1006,7 @@ void Settings::legacyLoad(const QString &path) {
 	LOAD(iTalkingUI_PrefixCharCount, "ui/talkingUI_PrefixCharCount");
 	LOAD(iTalkingUI_PostfixCharCount, "ui/talkingUI_PostfixCharCount");
 	LOAD(qsTalkingUI_AbbreviationReplacement, "ui/talkingUI_AbbreviationReplacement");
+	LOAD(iTalkingUI_TransparencyLevel, "ui/talkingUI_TransparencyLevel");
```

Comment:
Can be removed. This method loads legacy settings. Since you are creating a new setting, there will be no one with that setting trying to migrate.